src/components/Panel/Panel.js

```diff
@@ -29,6 +46,21 @@ class Panel extends Component {
           />
         )}
 
+        {collapsible && (
+          <button
```

Comment:
Do we want to make the button the only UI that controlled the collapsed state? e.g. we could make the complete header clickable (without losing a11y of course).

Reply:
It's a discussion worth triggering. It's not in the specs right now. We should first investigate if this is a pattern that is used elsewhere.

Reply:
IMO we should start with the button. More than that a custom panel could be made, although I'm not sure if that's very practical in just vanilla JS